CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Implemented file locking to avoid data races (guaranteed for Linux/MacOSX)
 - Implemented ```pull``` command to fetch and save multiple thing models at once
 - Implemented setting CORS options for API
+- Implemented fetching a TM by a \[partial\] semantic version or version digest also in REST API
```

Comment:
By "digest" should be removed from the comment, as it is not supported anymore.

Reply:
removed